Packs/Illumio/Integrations/IllumioCore/IllumioCore.py

```diff
@@ -70,10 +310,18 @@ def main():
         client.set_credentials(api_user, api_key)
 
         if command == 'test-module':
-            result = test_module(client)
+            return_results(test_module(client))
         else:
-            raise NotImplementedError(f'Command {command} is not implemented')
-        return_results(result)
+            ILLUMIO_COMMANDS = {
+                'illumio-traffic-analysis': illumio_traffic_analysis_command,
+                "illumio-virtual-service-create": illumio_virtual_service_create_command
```

Comment:
Please try to be consistent with double and single quotes.

Reply:
Yes